packages/Datadog.Unity/CHANGELOG.md

```diff
@@ -6,9 +6,15 @@
 * Added an option for detecting non-fatal app hangs within a given threshold on iOS.
 * Added documentation tooltips to the Datadog Options window (on Labels).
 * Added support for file / line mappings in C# exceptions through C# Native Stack Mapping
+* Make DatadogWorker's message pools thread safe.
+* Stop Resources in DatadogTrackedWebRequest if the underlying UnityWebRequest is Disposed.
```

Comment:
```suggestion
* Make `DatadogWorker`'s message pools thread safe.
* Stop resources in `DatadogTrackedWebRequest` if the underlying `UnityWebRequest` is disposed.
```

Reply:
🤔 I'm actually split on this. I can see why we might want them to be lowercase, but they are weirdly "proper" nouns referring to Datadog Resources and the specific Dispose method on C#....

Reply:
imo they should start from the capital letter if it is a reference to the code symbol (class, interface, etc.) and then be wrapped in the ticks.